ancestrymanager/ancestryutil.go

```diff
@@ -0,0 +1,96 @@
+package ancestrymanager
+
+import (
+	"fmt"
+	"strings"
+
+	resources "github.com/GoogleCloudPlatform/terraform-validator/converters/google/resources"
+)
+
+// AssetParent derives a resource's parent from its ancestors.
+func AssetParent(cai *resources.Asset, ancestors []string) (string, error) {
```

Comment:
Possibly name with the same convention as GetAncestors?

```suggestion
func GetParent(cai *resources.Asset, ancestors []string) (string, error) {
```

Reply:
the style guide says: Function and method names should not use a Get or get prefix, unless the underlying concept uses the word "get" (eg an HTTP GET)? 
I am going to remove the "Get" in the ancestor manager interface too.